projects/tomcat/HttpParserFuzzer.java

```diff
@@ -0,0 +1,10 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh;
+import org.apache.tomcat.util.http.parser.HttpParser;
+
+public class HttpParserFuzzer {
+    public static void fuzzerTestOneInput(FuzzedDataProvider data) {
```

Comment:
Ideally we'd like to see some more interesting targets :)

Reply:
@oliverchang there will be more tests integrated in the next week.

Reply:
Thank you for your feedback!

Reply:
OK, looking forward to it! Merging this for now.